automatilib/cola_auth/i_dot_ai_cola_backend.py

```diff
@@ -0,0 +1,33 @@
+import logging
+
+from django.contrib.auth import get_user_model
+from django.contrib.auth.backends import ModelBackend
+
+LOGGER = logging.getLogger(__name__)
+
+User = get_user_model()
+
+
+class AuthenticationBackend(ModelBackend):
+    def authenticate(self, request, username=None, password=None, **kwargs):
```

Comment:
```suggestion
    def authenticate(self, request, username: Optional[str[=None, password Optional[str[=None, **kwargs) -> User:
```

not 100% needed, but good to start adding annotations